src/ontology/cl-edit.owl

```diff
@@ -30820,6 +30834,119 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32555229") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4033015 "retinal astrocyte")
 EquivalentClasses(obo:CL_4033015 ObjectIntersectionOf(obo:CL_0000127 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000966)))
 
+# Class: obo:CL_4033016 (secondary crest myofibroblast)
+
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033016 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033016 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033016 "2023-03-16T09:17:47Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31331942") oboInOwl:hasExactSynonym obo:CL_4033016 "secondary-crest myofibroblast")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33624948") oboInOwl:hasRelatedSynonym obo:CL_4033016 "SCMF")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25448080") oboInOwl:hasRelatedSynonym obo:CL_4033016 "SCMFs")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31331942") oboInOwl:hasRelatedSynonym obo:CL_4033016 "alveolar myofibroblast")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31331942") Annotation(oboInOwl:hasDbXref "PMID:33624948") rdfs:comment obo:CL_4033016 "A myofibroblast that is part of an alveoli during alveolarization. The contractile force of this cell elongates the secondary crest, while producing a framework of elastin and tenascin. During the maturation of the septa, secondary crest myofibroblasts, together with matrix fibroblasts, secretes metalloproteinases and other ECM‐remodeling proteins to thin the septal tip ECM. The secondary crest myofibroblast continues producing elastin, eventually undergoing apoptosis during adulthood.")
+AnnotationAssertion(rdfs:label obo:CL_4033016 "secondary crest myofibroblast")
+SubClassOf(obo:CL_4033016 obo:CL_0000186)
+SubClassOf(obo:CL_4033016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002299))
```

Comment:
Based on cursory review of the references, unsure if this is accurate as they seem to be present before the formation of alveoli and described to be part of the "secondary crest" or septa. They also described as part of the mesenchyme/interstitium, which I think would be considered distinct from alveoli.

Reply:
It is very tricky, because it is present before the formation of the septa/second crest (it initiates their formation). I agree that it is present before the alveolar stage (saccular stage). Therefore, `'part of' some 'lung connective tissue'` and `'participates in' some 'lung alveolus development'` are more appropriate opiate.